File: JS/Card.js
class Card extends Payment {

    constructor(number, cvv, date) {
        super()
        this.number = number
        this.cvv = cvv
        this.date = date
    }

    //Polimorfismo
    printDataPayment() {
        super.printDataPayment()
        console.log("payment: Card")
        console.log("number: ", this.number, " cvv: ", this.cvv, " date: ", this.date)
    }
}

File: JAVA/Card.java
package JAVA;

public class Card extends Payment{
    
    int number;
    int cvv;
    String date;

    public Card(int number, int cvv, String date) {
        super();
        this.number = number;
        this.cvv = cvv;
        this.date = date;
    }

}
